integration/osrmfiles/tar_loader.go

```diff
@@ -25,7 +25,8 @@ func Load(contents ContentsOperator) error {
 			break // End of archive
 		}
 		if err != nil {
-			glog.Fatal(err)
+			glog.Errorf("parsing tar file %s failed, err: %v", contents.FilePath(), err)
+			break
```

Comment:
Who will handle partial failure, when part of data be load into writer?

Reply:
Yes. I have met issue that a file broked but some of the contents still possible to load, so just print a log and go with correct parts